.github/scripts/auto-backport.py

```diff
@@ -47,6 +47,11 @@ def create_pull_request(repo, new_branch_name, base_branch_name, pr, backport_pr
         )
         logging.info(f"Pull request created: {backport_pr.html_url}")
         backport_pr.add_to_assignees(pr.user)
+        if is_draft:
+            backport_pr.add_to_labels("conflicts")
+            pr_comment = f"@{pr.user} - This PR has conflicts, therefore it was moved to `draft` \n"
```

Comment:
This PR was marked as "draft" because it has conflicts\n

Reply:
fixed